### PR TITLE
github: remove issue templates

### PR DESCRIPTION
**Pull request by s-pace**
**Summary**

This PR removes the obsolete issues templates from algolia/docsearch

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!--
+  Thanks for participating in this project!
+  Please:
+    - do at least one GitHub search in current issues, maybe your question is already here
+-->
+
+## 📚 Documentation
```
Comment (francoischalifour) [inline .github/ISSUE_TEMPLATE.md]:
```suggestion
## Description
```

Did you mean "Description"? I don't understand "Documentation" as a title.

Comment (s-pace) [reply .github/ISSUE_TEMPLATE.md]: I did mean Documentation as this repo is mostly about Documentation

Comment (francoischalifour) [reply .github/ISSUE_TEMPLATE.md]: Then I don't think it makes sense as an issue title, it doesn't describe what users will have to fill.

Comment (s-pace) [reply .github/ISSUE_TEMPLATE.md]: I have updated the template.

**Review by francoischalifour — APPROVED** [review]

**Pull request merged**